src/index.html

```diff
@@ -13,12 +13,12 @@
     <link rel="stylesheet" href="styles/styles.css" />
   </head>
   <body>
-    <header class="app-header">
-      <h1 class="app-header__heading">Magic Quizard</h1>
-    </header>
     <main class="app-main">
+      <header class="app-header">
+        <h1 class="app-header__heading">Magic Quizard</h1>
+      </header>
```

Comment:
The `header` of the website should be outside of the main element. 
```suggestion
      <header class="app-header">
        <h1 class="app-header__heading">Magic Quizard</h1>
      </header>
      <main class="app-main">
```